index.html

```diff
@@ -0,0 +1,191 @@
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"/>
+    <title>Ade-Samuel Victor</title>
+</head>
+<body>
+    <header>
+        <div class="nav-logo">
+            <a href="index.html">Son of Zion</a>
+        </div>
+        <nav class="nav-button">
+            <a href="#">
+                <img src="./assets/images/Union.png" alt="hamburger-menu">
+            </a>
+        </nav>
+    </header>
+    <main>
+        <section class="intro-section">
+            <div class="intro">
+                <h1>I'm Victor<br>
+                    Glad to see<br> 
+                    you!
+                </h1>
+                <p>I'm a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, don't hestiate to contact me.
+                </p>
+                <h6>LET'S CONNECT</h6>
+                <ul class="social-icons">
+                    <li>
+                        <a href="https://www.twitter.com/son_ofzion"><img src="./assets/images/Vector-3.png" alt="twitter"></a>
+                    </li>
+                    <li>
+                        <a href="https://www.linkedin.com/in/sonofion"><img src="./assets/images/Linkedin-icon.png" alt="linkedin"></a>
+                    </li>
+                    <li>
+                        <a href="https://www.medium.com/@victoradesamuel16"><img src="./assets/images/Vector-1.png" alt="medium"></a>
+                    </li>
+                    <li>
+                        <a href="https://github.com/victoradesamuel"><img src="./assets/images/Vector-2.png" alt="github"></a>
+                    </li>
+                    <li>
+                        <a href="https://hashnode.com/@SonofZion"><img src="./assets/images/Vector.png" alt="hashnode"></a>
+                    </li>
+                </ul>
+            </div>
+        </section>
+        <section class="work-samples">
+            <div class="sample-1">
+                <img src="./assets/images/Snapshoot-Portfolio.png" alt="image">
+                <h3 class="title">Tonic</h3>
+                <div class="details">
+                    <div class="name">CANOPY</div>
+                    <div class="type">Back End Dev</div>
+                    <div class="year">2015</div>
+                </div>
+                <P class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</P>
+                <ul class="tools">
+                    <li class="html">html</li>
+                    <li class="css">css</li>
+                    <li class="javascript">javascript</li>
+                </ul>
+                <button class="see-project" type="button">See Project</button>
+            </div>
```

Comment:
Kindly adjust the `left` and `right` spacing for the work section to match the Figma design 
| Your design | Figma Design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/154268758-df687d8b-a11f-4117-b2c6-ff6f5813ca5b.png) | ![image](https://user-images.githubusercontent.com/24703171/154268872-302687ce-a863-45e2-a544-2b18b9cdc23c.png) |